### ISSUE TITLE
How to transform the datasets?

### ISSUE THREAD

**Issue by grapefruitL** [vahidk/tfrecord#12]
Hi! It's nice to u to provide this,but i still have some problems
i see the code in datasets.py:

`worker_info = torch.utils.data.get_worker_info()
        if worker_info is not None:
            shard = worker_info.id, worker_info.num_workers
            np.random.seed(worker_info.seed % np.iinfo(np.uint32).max)
        else:
            shard = None
        it = reader.tfrecord_loader(
            self.data_path, self.index_path, self.description, shard)
        if self.shuffle_queue_size:
            it = iterator_utils.shuffle_iterator(it, self.shuffle_queue_size)`

does 'iterator_utils' has other methods to do transform? like calling map function in tensorflow? if does, where can i find more information about how to use this tfrecord

**Comment by vahidk** [comment]
This is a pytorch question. You do the transform like any other dataset in your loader.
